next.config.js

```diff
@@ -160,4 +160,12 @@ const sentryWebpackPluginOptions = {
   // https://github.com/getsentry/sentry-webpack-plugin#options.
 };
 
-export default withSentryConfig(nextConfig, sentryWebpackPluginOptions)
+const sentryOptions = {
+  // Upload additional client files (increases upload size)
+  widenClientFileUpload: true,
+
+  // Hides source maps from generated client bundles
+  hideSourceMaps: true,
```

Comment:
I don't think we need to hide these, and as an open source org, I actually like making our source code more accessible :)

```suggestion
```

Reply:
Yeah agree, no point in hiding, the source is open anyway. Having stuff minified for perf reasons is great but no obfuscation intended :)

Reply:
So I think it's actually hidden by default (true by default). Maybe we should set it as false